api/src/main/java/net/md_5/bungee/api/boss/BossBar.java

```diff
@@ -0,0 +1,159 @@
+package net.md_5.bungee.api.boss;
+
+import java.util.Collection;
+import net.md_5.bungee.api.chat.BaseComponent;
+import net.md_5.bungee.api.connection.ProxiedPlayer;
+
+/**
+ * Represents a boss bar object
+ */
+public interface BossBar
+{
+
```

Comment:
hasPlayer and hasFlag should be included

Reply:
Ok

Reply:
Yes but these will both create one time list copies just to check if they contain an element which can be avoided.